### ISSUE TITLE
Should 'graphClient.Me' being used with application Application Mail.Send permission?

### ISSUE THREAD

**Issue by lertoo** [microsoftgraph/microsoft-graph-docs#7813]
I created an authProvider for the code above:

IConfidentialClientApplication app = ConfidentialClientApplicationBuilder
				.Create(&lt;clientID&gt;)
				.WithClientSecret(&lt;secret&gt;)
				.WithTenantId(&lt;tenantID&gt;)
				.Build();
ClientCredentialProvider authProvider = new ClientCredentialProvider(app);

But it cause the following error:
System.AggregateException: 'One or more errors occurred. (Code: BadRequest
Message: Current authenticated context is not valid for this request. This occurs when a request is made to an endpoint that requires user sign-in. For example, /me requires a signed-in user.  Acquire a token on behalf of a user to make requests to these endpoints.  Use the OAuth 2.0 authorization code flow for mobile and native apps and the OAuth 2.0 implicit flow for single-page web apps.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d3b984b6-9100-69aa-f0f8-84bcdcea20dd
* Version Independent ID: 3f8b622d-becd-f287-98ac-18e408200523
* Content: [Send mail - Microsoft Graph v1.0](https://docs.microsoft.com/en-us/graph/api/user-sendmail?view=graph-rest-1.0&tabs=csharp#feedback)
* Content Source: [api-reference/v1.0/api/user-sendmail.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/v1.0/api/user-sendmail.md)
* Product: **outlook**
* Technology: **microsoft-graph**
* GitHub Login: @svpsiva
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by lertoo** [comment]
I just want to implement the ASP.NET Core Identity IEmailSender with Graph API, for sending emails from the app for password reset etc. But the documentation is quite hard to follow.

**Comment by svpsiva** [comment]
Hi

Can you try using 

graphClient.Users[userId]
	.SendMail(message,null)
	.Request()
	.PostAsync();

**Comment by lertoo** [comment]
Thanks svpsiva. It works. 
Hope the docs will be updated to help others.

> Hi
> 
> Can you try using
> 
> graphClient.Users[userId]
> .SendMail(message,null)
> .Request()
> .PostAsync();
